pytubefix/helpers.py

```diff
@@ -333,7 +334,45 @@ def create_mock_html_json(vid_id) -> Dict[str, Any]:
     return html_data
 
 
-# Remove ANSI color codes from a colored string
 def strip_color_codes(input_str):
+    """Remove ANSI color codes from a colored string"""
     ansi_escape = re.compile(r'\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])')
     return ansi_escape.sub('', input_str)
+
+
+def reset_cache(verbose: bool = False):
+    """
+    Deletes the `__cache__` directory to reset the cache.
+
+    This function checks if the `__cache__` directory exists in the same directory 
+    as the script. If it exists and is a directory, it deletes it along with its contents. 
+    If the directory does not exist, it logs a message indicating that the cache directory 
+    is not present.
+
+    Parameters:
+        verbose (bool): If True, sets up logging at the DEBUG level. Default is False.
+
+    Behavior:
+        - When `verbose` is True, debug messages are logged, providing information about 
+          the existence and status of the cache directory.
+        - If `verbose` is False, no logging configuration is modified, and debug messages 
+          may not be shown unless logging is already configured externally.
+
+    Example:
+        >>> reset_cache(verbose=True)
+        # Logs detailed information about the cache reset process.
+
+    Raises:
+        None
+    """
+    
+    cache_dir = os.path.join(os.path.dirname(__file__), '__cache__')
+
+    if verbose:
+        setup_logger(level=logging.DEBUG)
+
```

Comment:
I think that specifying verbose is not necessary, we can get all the logs using for example: 

```python
logging.basicConfig(level=logging.DEBUG, format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
```

Reply:
I'll keep it like this, by default it won't show anything